Valet/VALValet.m

```diff
@@ -608,8 +608,9 @@ - (nonnull NSString *)secServiceIdentifier;
 - (nullable NSString *)_sharedAccessGroupPrefix;
 {
     NSDictionary *query = @{ (__bridge NSString *)kSecClass : (__bridge NSString *)kSecClassGenericPassword,
-                             (__bridge id)kSecAttrAccount : @"SharedAccessGroupPrefixPlaceholder",
-                             (__bridge id)kSecReturnAttributes : @YES };
+                             (__bridge id)kSecAttrAccount : @"SharedAccessGroupAlwaysAccessiblePrefixPlaceholder",
+                             (__bridge id)kSecReturnAttributes : @YES,
+                             (__bridge id)kSecAttrAccessible : (__bridge id)kSecAttrAccessibleAlwaysThisDeviceOnly };
```

Comment:
accessible always since this bit of data is not a secret, and we need to be able to access it always.